src/crimson/tools/store-nbd.cc

```diff
@@ -0,0 +1,644 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:nil -*-
```

Comment:
:+1:  personally, i think we should just use spaces in future. much easier when reviewing changes.

Reply:
Most of the code base does this.